web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@alejandrosantana, you have made it right. Include the source SVG when the icon is custom